package-lock.json

```diff
@@ -10808,7 +10813,9 @@
       "version": "8.1.2",
       "resolved": "https://registry.npmjs.org/superagent/-/superagent-8.1.2.tgz",
       "integrity": "sha512-6WTxW1EB6yCxV5VFOIPQruWGHqc3yI7hEmZK6h+pyk69Lk/Ut7rLUY6W/ONF2MjBuGjvmMiIpsrVJ2vjrHlslA==",
+      "deprecated": "Please upgrade to v9.0.0+ as we have fixed a public vulnerability with formidable dependency. Note that v9.0.0+ requires Node.js v14.18.0+. See https://github.com/ladjs/superagent/pull/1800 for insight. This project is supported and maintained by the team at Forward Email @ https://forwardemail.net",
```

Comment:
Sure, this was the version that was previously installed but since this warning appeared now that it's explicit, I think it's worth updating it. Done in https://github.com/HathorNetwork/hathor-wallet-headless/pull/512/commits/33248c9f10a69b33d8c3a8461698f6d8bb8faa09, let's see if all tests will pass. I updated `supertest` to its latest version (`7.0.0`), and `superagent` to the resolved version required by `supertest` (`^9.0.1` which resolved to `9.0.2`).